automation/submission-perf-test.sh

```diff
@@ -9,13 +9,33 @@ WORKING_DIR=${3:-$PWD}
 
 # Check if $ENV is not empty
 if [[ ! ${ENV} ]]; then
-    echo "ENV is empty. Run again \`sh submission-perf-test.sh <alpha or staging>\`"
+    echo
+    "ENV is empty. Run again sh submission-perf-test.sh <alpha or staging"
     exit 1
-else echo "Starting Perf test in $ENV"; fi
-
+else
+    echo "Starting Perf test in {$ENV}"
+fi
 
 JSON_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN -e VAULT_ADDR=https://clotho.broadinstitute.org:8200 broadinstitute/dsde-toolbox vault read -format=json secret/dsde/firecloud/dev/common/firecloud-account.pem | jq '.data'`
 
+callbackToNIH() {
+    user=$1
+
+    echo "
+    Launching calback to NIH for:
+    user=$1
+    "
+     ACCESS_TOKEN=`docker run --rm -v $WORKING_DIR:/app/populate -w /app/populate broadinstitute/dsp-toolbox python get_bearer_token.py "${user}" "${JSON_CREDS}"`
+
+    # Verify that user does not need to refresh their token
+        if
```

Comment:
This block of code for refresh tokens is duplicated in this script. Delete one of them. It makes sense to check all access issues before running the script so delete the other block. It's also not related to the NIH call so either it shouldn't be in this function or it should be renamed.

Reply:
Don't we need to get access token for the user?

Reply:
(we face talked -- this is for the refresh token)